5-WebApp-AuthZ/5-2-Groups/README.md

```diff
@@ -49,521 +71,385 @@ From your shell or command line:
-   - Click on the **Add permissions** button at the bottom.
-1. At this stage permissions are assigned correctly and the **GroupMember.Read.All** requires admin to consent.
-   Click the **Grant/revoke admin consent for {tenant}** button, and then select **Yes** when you are asked if you want to grant consent for the
-   requested permissions for all account in the tenant.
-   You need to be an Azure AD tenant admin to do this.
-
-##### Configure the  web app (WebApp-GroupClaims) to use your app registration
-
-Open the project in your IDE (like Visual Studio) to configure the code.
->In the steps below, "ClientID" is the same as "Application ID" or "AppId".
-
-1. Open the `appsettings.json` file
-1. Find the app key `ClientId` and replace the existing value with the application ID (clientId) of the `WebApp-GroupClaims` application copied from the Azure portal.
-1. Find the app key `TenantId` and replace the existing value with your Azure AD tenant ID.
-1. Find the app key `Domain` and replace the existing value with your Azure AD tenant name.
-1. Find the app key `ClientSecret` and replace the existing value with the key you saved during the creation of the `WebApp-GroupClaims` app, in the Azure portal.
-
-#### Configure your application to receive the **groups** claim
-
-In the sample, a dashboard component allows signed-in users to see the tasks assigned to them or other users based on their memberships to one of the two security groups, **GroupAdmin** and **GroupMember**. Please use the instructions provided at [Create a basic group and add members using Azure AD](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal) to create these security groups, if not available already. 
+    1. In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `WebApp-GroupClaims`.
+    1. Under **Supported account types**, select **Accounts in this organizational directory only**
+    1. Select **Register** to create the application.
+1. In the **Overview** blade, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. In the app's registration screen, select the **Authentication** blade to the left.
+1. If you don't have a platform added, select **Add a platform** and select the **Web** option.
+    1. In the **Redirect URI** section enter the following redirect URIs:
+        1. `https://localhost:44321/`
+        1. `https://localhost:44321/signin-oidc`
+    1. In the **Front-channel logout URL** section, set it to `https://localhost:44321/signout-oidc`.
+     1. In the **Implicit grant** section, check the **ID tokens** option as this sample requires the [Implicit grant flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-implicit-grant-flow) to be enabled to sign-in the user and call an API.
```

Comment:
>   1. In the **Implicit grant** section, check the **ID tokens** option as this sample requires the [Implicit grant flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-implicit-grant-flow) to be enabled to sign-in the user and call an API.

This is a bit misleading as just checking the ID token is called hybrid flow, not implicit. If the codegen generates this block, we should probably update it? cc @kalyankrishna1

Reply:
The issue is CodeGen prints out the instruction to enable implicit flow twice one time mentioning the `implicit flow` the other time just instructing to enable it.

I'll remove the duplicate but yes we'll need to come back to this.